internal/compose/compose.go

```diff
@@ -340,7 +345,29 @@ func (p *Project) runDockerComposeCmd(opts dockerComposeOptions) error {
 	return cmd.Run()
 }
 
+func (p *Project) DockerComposeVersion() (string, error) {
+
+	const (
+		v1marker = "docker-compose version 1"
```

Comment:
Please move it to the file const section.

Could you please also add a comment with a full dump of `docker-compose version`, so it's clear for the next developer what's going on and why.

Reply:
Same story: it isn't resolved.